packages/taquito-rpc/test/data/rpc-responses.ts

```diff
@@ -5404,6 +5404,109 @@ export const ticketBalancesResponse = [
   },
 ];
 
+export const pendingOperationsResponse = {
```

Comment:
Would be nice if the team add type check on `createRequest` as it merely does a conversion from `any` to any type of the template is provided.

As `createRequest` lacks type check I typed this constant as `MempoolPendingOperationsResponse` to enforce the test to do it.

May you add the `MempoolPendingOperationsResponse` type back?

Reply:
I renamed type `MempoolPendingOperationsResponse` to `PendingOperations` in packages/taquito-rpc/src/types.ts line 2138 to separate it from the mock rpc response aka pendingOperationsResponse here. And have pushed a commit to use the type on the rpc call in tests to enforce it.

Reply:
Perfect! 👍